app/models/school.rb

```diff
@@ -27,6 +31,10 @@ def full_address
     address.squeeze("\n")
   end
 
+  def full_address_formatted
```

Comment:
I'd recommend adding a presenter and moving this method as well as :full_address there. I prefer the logic in the models to be business like / related to the subject domain rather than just presentation.

Reply:
Do we have any presenter/decorator patterns?

Reply:
No yet, we can always add one though, draper, delegator

Reply:
Done